.github/workflows/sca.yml

```diff
@@ -83,10 +83,13 @@ jobs:
       - name: Check - composer-require-checker
         run: ./dev-tools/vendor/bin/composer-require-checker check composer.json --config-file .composer-require-checker.json
 
+      - name: Check - composer bump
+        if: ${{ github.event_name == 'pull_request' }}
```

Comment:
please add a comment, otherwise it's not obvious why we do it
(example: comment for `name: Unlock dev-tools`)